pylib/src/lib.rs

```diff
@@ -9,6 +9,10 @@ pub const LIB_PATH: &str = match option_env!("win_lib_path") {
     None => concat!(env!("CARGO_MANIFEST_DIR"), "/Lib"),
 };
 
-#[cfg(feature = "freeze-stdlib")]
+#[cfg(feature = "freeze-stdlib", not(target_os = "windows"))]
 pub const FROZEN_STDLIB: &rustpython_compiler_core::frozen::FrozenLib =
     rustpython_derive::py_freeze!(dir = "./Lib", crate_name = "rustpython_compiler_core");
+
+#[cfg(feature = "freeze-stdlib", target_os = "windows")]
+pub const FROZEN_STDLIB: &rustpython_compiler_core::frozen::FrozenLib =
+    rustpython_derive::py_freeze!(dir = "../Lib", crate_name = "rustpython_compiler_core");
```

Comment:
This will cause problem for cargo publish.

Reply:
Ah ... that explains a lot, however it was already broken on windows coming in. I'll see if I can fix the symlink on windows in that case.

Reply:
There probably was a script written by @coolreader18 to copy them. 
Otherwise now windows also has symlink.
If everything cause problem, I vote for moving them under pylib than this change

Reply:
Suggestion: A good first step to resolve this error is adding a CI job to reproduce this problem